sql/updates/world/master/9999_99_99_19345_world.sql

```diff
@@ -2,6 +2,6 @@ DELETE FROM `spell_areatrigger` WHERE (`SpellMiscId`=5420 AND `AreaTriggerId`=10
 INSERT INTO `spell_areatrigger` (`SpellMiscId`, `AreaTriggerId`, `MoveCurveId`, `ScaleCurveId`, `MorphCurveId`, `FacingCurveId`, `DecalPropertiesId`, `TimeToTarget`, `TimeToTargetScale`, `VerifiedBuild`) VALUES
 (5420, 10133, 0, 0, 0, 0, 0, 0, 7177, 23420); -- SpellId : 5740
 
-DELETE FROM `spell_script_names` WHERE `spell_id`=5740;
+DELETE FROM `spell_script_names` WHERE `spell_id`=5740 OR `ScriptName`="spell_warl_rain_of_fire";
```

Comment:
Well, if you think about the use of `AND`, it is very selective whereas `OR` is not.
This variant opens for deleting both for all cases of `ScriptName`='spell_warl_rain_of_fire'
as well as deleting that special case of `spell_id`= 5740.